plugin/pkg/scheduler/schedulercache/node_info.go

```diff
@@ -361,6 +366,14 @@ func calculateResource(pod *v1.Pod) (res Resource, non0_cpu int64, non0_mem int6
 		non0_mem += non0_mem_req
 		// No non-zero resources for GPUs or opaque resources.
 	}
+
+	//Account for storage requested by emptydir volumes
+	for _, vol := range pod.Spec.Volumes {
+		if vol.EmptyDir != nil {
+			res.StorageScratch += vol.EmptyDir.SizeLimit.Value()
```

Comment:
it would be good if we have a utility functoin for this as this loop is repeated in multiple places.

also need to ignore medium memory empty dir.